src/mson.js

```diff
@@ -173,24 +173,26 @@ function handle(name, element, { parent = null, spaces = 4, marker = '+',
     str += ` ${name}`;
   }
 
-  // Next, comes the optional example value
-  if (element.content && typeof element.content !== 'object') {
-    if (parent && parent.element !== 'array') {
-      str += ':';
-    }
+  if (element) {
```

Comment:
When a member of an object doesn't have a value (`element` in this scope). Element will be undefined below.